doc/manual.tex

```diff
@@ -544,9 +544,9 @@ \chapter{Acknowledgements and history}
 separate library for general use and improvement.  The namespace was
 changed from NR to Geom and a prototype for paths sketched out.
 Nathan studied the state of the art for computational geometry whilst
-mental focussed on the design of Paths.
+mentally focused on the design of Paths.
```

Comment:
"mental" likely refers to "MenTaLguY" here, one of the early Inkscape devs.

Reply:
oh ok! Should i change it to `MenTaLguY` then ?

Reply:
Ahh good catch Ede123.  Yes, change it to MenTaLguY to avoid confusion.  He was heavily involved in the early 2geom development so you may want to scan around for other instances of that word that may be worth clarifying.

Reply:
Swapped out 92f2955